### PR TITLE
circle ci

### PR DESCRIPTION
**Pull request by calvix**
circleCI integration

### CODE REVIEW

**Comment by calvix** [comment]
ping @JosephSalisbury

File: README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# mayuctl
+tools that was previously included in https://github.com/giantswarm/mayu
+
+
+is used for listing or chaning machines registered to `mayu`
```
Comment (JosephSalisbury) [inline README.md]: chaning is not a word <3

File: README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+```
```
Comment (JosephSalisbury) [inline README.md]: I'd just get rid of this. It's gonna go out of date super quickly.

**Review by JosephSalisbury — APPROVED** [review]
Small readme notes. Go ahead as you see fit

**Comment by JosephSalisbury** [comment]
Didn't see the failing build. Hold on.

**Pull request merged**